### ISSUE TITLE
Metrics aggregation review

### ISSUE THREAD

**Issue by yorugac** [grafana/k6-operator#192]
k6 have been using partial metrics aggregation for k6 Cloud processing, best explained here:
https://github.com/grafana/k6/blob/b750e206859a495b8a1fe12aa016dc8ed0e1c9db/cloudapi/config.go#L40-L96

The k6-operator cloud output run does not take this into account at the moment. Find out:
- did the new time series in k6 impact this partial aggregation?
- what is the most performant way to use cloud output in k6 at the moment?
- what needs to be changed in k6-operator invocation of k6 to take advantage of performant aggregation?

Implement changes as needs be.

**Comment by yorugac** [comment]
Metrics aggregation can be turned on k6's side with these env vars:
```
  K6_CLOUD_AGGREGATION_MIN_SAMPLES
  K6_CLOUD_AGGREGATION_PERIOD
  K6_CLOUD_AGGREGATION_WAIT_PERIOD
  K6_CLOUD_METRIC_PUSH_INTERVAL
  K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE
  K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY
```

But also watch out for the progress in https://github.com/grafana/k6/issues/2954: once it is resolved, aggregation may be turned on by default, without the need for any env vars tinkering.

cc @codebien

**Comment by yorugac** [comment]
This is considered resolved with PR #229
